Update mul int16 test

3l1 opened the pull request:
Differential Revision: D83437473

digantdesai reviewed (approved):
Review automatically exported from Phabricator review in Meta.

3l1 committed:
Update mul int16 test for Ethos (#14646)

Summary:


bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: digantdesai

Differential Revision: D83437473
